packages/react/src/components/ComboChartCard/comboChartHelpers.js

```diff
@@ -188,6 +188,7 @@ export const useChartOptions = (content) => {
       size,
       timeDataSourceId,
       tooltipDateFormatPattern,
+      defaultDateFormatPattern,
```

Comment:
we should set the default like we do in the tableCardUtils file but using the old format, 'L HH:mm'

Reply:
Done